nixos/modules/services/networking/ssh/sshd.nix

```diff
@@ -12,8 +12,23 @@ let
     then cfgc.package
     else pkgs.buildPackages.openssh;
 
+  /* reports boolean as yes / no and dont use the "=" operator
+  */
+  mkValueStringSshd = v:
+        if isInt           v then toString v
+        else if isString   v then v
+        else if true  ==   v then "yes"
+        else if false ==   v then "no"
+        else if isSecret   v then hashString "sha256" v._secret
```

Comment:
Function isSecret is not defined in this file. For which ssh configuration should we use this type?

Reply:
it was a bad copy/paste, I removed its usage.